packages/editor-ui/src/constants.ts

```diff
@@ -458,6 +458,13 @@ export const enum VIEWS {
 	WORKER_VIEW = 'WorkerView',
 }
 
+export const EDITABLE_VIEWS = [
```

Comment:
Suggestion - `EDITABLE_CANVAS_VIEWS` or maybe we can just set these directly in the routes files..